electron/src/lib/CoreProtocol.ts

```diff
@@ -36,61 +36,74 @@ const CORE_PROTOCOL_MAX_LENGTH = 1024;
 
 enum ProtocolCommand {
   SHOW_CONVERSATION = 'conversation',
+  SHOW_USER = 'user',
   START_SSO_FLOW = 'start-sso',
 }
 
-const dispatcher = async (url?: string) => {
-  if (typeof url === 'undefined' || !url.startsWith(`${CORE_PROTOCOL}://`) || url.length > CORE_PROTOCOL_MAX_LENGTH) {
-    return;
-  }
+export class CustomProtocolHandler {
+  public hashLocation: string = '';
+
+  async dispatcher(url?: string) {
```

Comment:
I would rename this function to `dispatch` instead of `dispatcher` for semantic purposes :)